tests/integration/security/file_mounted_certs/main_test.go

```diff
@@ -0,0 +1,204 @@
+// +build integ
+//  Copyright Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package filemountedcerts
+
+import (
+	"context"
+
+	"io/ioutil"
+	"path"
+	"strings"
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/pkg/test/framework/resource"
+
+	"istio.io/istio/pkg/test/env"
+	"istio.io/istio/pkg/test/framework/components/environment/kube"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	inst istio.Instance
+)
+
+const (
+	ProxyMetadataJson = `
```

Comment:
I would avoid adding things other than exactly what we want to test. This makes it less confusing what the intent of the test is, and what are required and "optional" settings.

Namely, jwt policy, xds via agent, dns capture, and stats settings

Reply:
```
"PROXY_XDS_VIA_AGENT": "false",
```
was needed due to the fact that i reuse existing test certificates in this test and they don't have enough info.
without this config i was seeing the following error:

```
	info	Could not verify certificate: peer certificate does not contain 1 URI type SAN, detected 0
```

Going to add proper test certs and that will allow to remove most of the custom settings.